modules/ocf_irc/manifests/znc.pp

```diff
@@ -0,0 +1,28 @@
+# A minimal ZNC installation
+# All extra configuration is managed by web interface and kept in backups
+class ocf_irc::znc {
+  package { 'znc': }
+
+  user { 'znc':
```

Comment:
call it ocfznc? i think we should be naming all users we create starting with `ocf` when possible to avoid conflicts with ldap or debian packaging